### ISSUE TITLE
Add security auditing JavaScript

### ISSUE THREAD

**Issue by dmohs** [DataBiosphere/saturn-ui#882]
Bernick asked for this. I judge it to be a low-impact way for us to add to our compliance package. Let me know if you don't agree.

One for each environment:
```html
<script src="https://jsagent.tcell.io/tcellagent.min.js" tcellappid="saturnnonprod-O9lUP" tcellapikey="AQEBBAEkx4iE2KxNyI7Wx08EwU1ycTM7E4FMSmaibbMUQxNU6uQvuAJt7fyABAtFYSYfgEE"></script>
```
```html
<script src="https://jsagent.tcell.io/tcellagent.min.js" tcellappid="saturnprod-VuHKy" tcellapikey="AQEBBAEGP2gTM2pIdJAQOIeNrm8dcTM7E4FMSmaibbMUQxNU6qy6nLPOBK8QfSvPFSsX8PQ"></script>
```

**Comment by bradtaylor** [comment]
It adds additional security auditing

**Comment by kleighwe** [comment]
@zarsky-broad  will make a separate ticket for prep work for this ticket.

**Comment by kleighwe** [comment]
BLOCKED on Isaac's ticket to be made :)
